eu.cichy1173.tabela.yml

```diff
@@ -0,0 +1,61 @@
+app-id: eu.cichy1173.tabela
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: tabela
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
```

Comment:
this is a very wide filesystem permission.

Why is it necessary?

Reply:
Main goal of app is opening csv files for reading content in tables. User also can add new rows and save file as csv or xml.

Reply:
You are right. It is to much wide. I changed it for home and I removed `network`.

I think, home is the best choice. Why? Main goal of the app is opening users csv files for displaying content in table, and also adding new rows and saving in xml or csv.